src/Goerli-DssSpellCollateral.sol

```diff
@@ -30,58 +68,180 @@ contract DssSpellCollateralAction {
     //
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
-    //
+
+    uint256 constant ZERO_PCT_RATE                  = 1000000000000000000000000000;
+
+    // --- Math ---
+    uint256 public constant WAD = 10**18;
+    uint256 public constant RAY = 10**27;
+    uint256 public constant RAD = 10**45;
+
+    // -- RWA007 MIP21 components --
+    address constant RWA007                         = 0x02091C59C92fc5E0575b3B133c3caa4f57D79b0e;
```

Comment:
wondering why `RWA007` was chosen as it should refers to SolarX.

Reply:
SolarX is not going to be onboarded.
@godsflaw argued that we should probably "fill the gap" with RWA007, since we skipped it in the past.